azure-mgmt-trafficmanager/src/main/java/com/microsoft/azure/management/trafficmanager/TrafficManagerEndpoint.java

```diff
@@ -225,6 +227,31 @@
         }
 
         /**
+         * The stage of the traffic manager endpoint definition allowing to specify the geographic region.
+         *
+         * @param <ParentT> the return type of {@link WithAttach#attach()}
+         */
+        interface WithGeographicLocation<ParentT> {
+            /**
+             * Specifies the geographic location for the endpoint that will be used when the parent profile is configured
+             * with Geographic based routing method {@link TrafficRoutingMethod#GEOGRAPHIC}.
+             *
+             * @param geographicLocation the geographic location
+             * @return the next stage of the definition
+             */
+            WithAttach<ParentT>  withGeographicLocation(String geographicLocation);
```

Comment:
for new methods/interfaces, let's make sure to annotate with `@Beta(SinceVersion.V1_2_0)`. Alva is about to update the annotations library with the new value.